src/OmniSharp.Abstractions/Models/v2/QuickInfoResponse.cs

```diff
@@ -0,0 +1,35 @@
+﻿#nullable enable
+namespace OmniSharp.Models.v2
+{
+    public class QuickInfoResponse
+    {
+        /// <summary>
+        /// Description of the symbol under the cursor. This is expected to be rendered as a C# codeblock
+        /// </summary>
+        public string? Description { get; set; }
+
+        /// <summary>
+        /// Documentation of the symbol under the cursor, if present. It is expected to be rendered as markdown.
+        /// </summary>
+        public string? Summary { get; set; }
```

Comment:
I'm not sure `Description` and `Summary` are really required here, and we can just reduce it down to a list of sections.

When looking at the [apis for vscode](https://code.visualstudio.com/api/references/vscode-api#Hover) or the [lsp spec](https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_hover) they either take in one markdown block or a set of them.

There is no specific ordering, so Summary and description should be the first blocks of course.

Reply:
@filipw @JoeRobich @mholo65 thoughts?

Reply:
simpler is better indeed, would make it more straightforward to expose via the LSP too

Reply:
Agree, since VS Code and LSP API expects a single block, or block of text, there's no need for separate Summary and Description. Just build everything into the same string.

Reply:
It can't be a single string. I can do this as a series of sections, but vscode's markdown string doesn't actually respect triple-backtick with a language. You have to actually call `addCodeblock` with a specific language.

Reply:
I just realized that I was making some beginner javascript mistakes when I was testing the single-string support last night, and I can do better. I'll rewrite this to be a single string.

Reply:
Updated this to send just a pre-formatted markdown string.